Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,66 @@
 ﻿using System;
+using System.Text.RegularExpressions;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Regex nameRegex = new Regex("^[aA-zZ]{2,100}$");
```

Comment:
Regex is truly an overkill! Often, especially in job interviews, you get negatives points for doing more than is asked. For example, edge case:
Tim O'conel. Rip 😄 
It's cool that you did this, but it's too easy to get it wrong.